components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_i64_to_string(path: &str, json: &str) -> String;
+      fn convert_string_to_i64(path: &str, json: &str) -> String;
+    }
+}
+
+// Converts int64/uint64 value to string and serializes json with new values.
```

Comment:
`Converts int64/uint64 value` this only handles uint, right? Not int?

Reply:
both

Reply:
refactored to have different functions